osu.Server.Spectator/Hubs/ILoggableHub.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Logging;
+
+namespace osu.Server.Spectator.Hubs
+{
+    internal interface ILoggableHub
```

Comment:
don't like the name, sounds wrong - this is a hub that can log (i.e. an `ILoggingHub`, or an `IHubWithLogging`, or even more abstractly an `ICanLog` or an `IHasLogger`), not a log that is *being* logged (which `ILoggableHub` implies)

Reply:
Was unsure about the name, funny enough I had `ILoggingHub` at first. Will rename to that.